io.github.diegoivan.pdf_metadata_editor.json

```diff
@@ -0,0 +1,84 @@
+{
+  "app-id": "io.github.diegoivan.pdf_metadata_editor",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "43",
+  "sdk": "org.gnome.Sdk",
+  "command": "pdf-metadata-editor",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "/share/vala",
+    "*.la",
+    "*.a"
+  ],
+  "modules": [
+    {
+      "name": "libportal",
+      "buildsystem": "meson",
+      "config-opts": [
+        "-Dbackend-gtk3=disabled",
+        "-Dbackend-gtk4=enabled",
+        "-Dbackend-qt5=disabled",
+        "-Ddocs=false",
+        "-Dtests=false"
+      ],
+      "sources": [
+        {
+          "url": "https://github.com/flatpak/libportal.git",
+          "type": "git",
+          "commit": "09ec040159b754c14610ad02e4de17d0b9bc6612"
+        }
+      ]
+    },
+
+    {
+      "name": "poppler",
+      "buildsystem": "cmake",
+      "config-opts": [
+        "-DENABLE_LIBOPENJPEG:STRING=none",
+        "-DBUILD_GTK_TESTS=OFF",
+        "-DBUILD_QT5_TESTS=OFF",
+        "-DBUILD_QT6_TESTS=OFF",
+        "-DBUILD_CPP_TESTS=OFF",
+        "-DBUILD_MANUAL_TESTS=OFF",
+        "-DENABLE_UTILS=OFF",
+        "-DENABLE_CPP=OFF",
+        "-DENABLE_QT5=OFF",
+        "-DENABLE_QT6=OFF",
+        "-DENABLE_BOOST=OFF"
```

Comment:
I'll assume you don't need the splash backend for poppler.

Reply:
No, the app does not need it.